SubmittingPatches-backports.rst

```diff
@@ -326,6 +326,12 @@ editing. If you need to add additional information to the cherry-pick commit
 message, append that information below this line. Once again: do not modify the
 original commit message.
 
+In order to get the PR created by the script, go back to master to run the same
+command or run the following command with your previous used arguments to execute
+the current script::
+
+    bash <(curl -Ls https://github.com/ceph/ceph/raw/master/src/script/ceph-backport.sh) $args
```

Comment:
This should never be necessary, since one should only ever run the latest version of `ceph_backport.sh` from the master branch.

Maybe we could add this curl shortcut as a quick way to update one's local copy of `ceph_backport.sh`, somewhere above where the doc says:

    The script is located at src/script/ceph-backport.sh in the master branch. 
    Though there might be an older version of this script in a stable branch, 
    do not use it. Only use the most recent version from master.

What do you think?